network-store-server/src/test/java/com/powsybl/network/store/server/CustomCassandraUnitTestExecutionListener.java

```diff
@@ -5,12 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 package com.powsybl.network.store.server;
```

Comment:
This file has to be removed?

Reply:
why would we keep something linked with cassandraUnit when we removed it from our code.